src/main/java/rx/Completable.java

```diff
@@ -1081,6 +1081,21 @@ public final Completable compose(CompletableTransformer transformer) {
     }
     
     /**
+     * Returns an Observable which will subscribe to this Completable and once that is completed then 
+     * will subscribe to the {@code next} Observable. An error event from this Completable will be 
+     * propagated to the downstream subscriber and will result in skipping the subscription of the 
+     * Observable.  
+     * 
+     * @param next the Observable to subscribe after this Completable is completed, not null
+     * @return Observable that composes this Completable and next
+     * @throws NullPointerException if next is null
+     */
+    public final <T> Observable<T> andThen(Observable<T> next) {
+        requireNonNull(next);
+        return next.delaySubscription(toObservable());
```

Comment:
One initial motivation for `andThen` was based on the [Option::and_then method] (https://doc.rust-lang.org/std/option/enum.Option.html#method.and_then) in Rust stdlib. It chains through the computation and transforms it's type/contents to the next Option.